### DISCUSSION TITLE
Import tags from image files

### DISCUSSION THREAD

**Discussion by DarthChoco** [photoprism/photoprism#3987] [General]
Is there any way to import tags that i wrote with Adobe Bridge to all of my image files? Took me a lot of time to write especially the names of the persons and actual i see no chance to see these values in photoprism.

**Comment by graciousgrey** [comment]
PhotoPrism does already read a lot of metadata tags. You can find the full list here:

https://www.photoprism.app/kb/metadata

Reading face tags from other tools is planned for the future, as is support for hierarchical tags.
